src/parser/bison_parser.y

```diff
@@ -752,12 +777,16 @@ opt_top:
 	;
 
 opt_limit:
-		LIMIT expr { $$ = new LimitDescription($2, nullptr); }
+	limit { $$ = $1; }
+	|	/* empty */ { $$ = nullptr; }
+	;
+
+limit:
```

Comment:
Well yeah, that was a left over from one of our first tries that I did not remove for this pr. TBH I almost feel like it would help the understanding for people to have a difference here. Will talk about it with our TA on wednesday

Reply:
Fixed in new patch